source/_posts/2017-11-15-code-generation-with-sourcery.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Getting stack traces in Swift on Linux
+date: 2017-11-15 13:00:00
```

Comment:
Just a reminder - before going live with the post we should remember to update this date